internal/proto/streaming.proto

```diff
@@ -13,7 +13,7 @@ import "google/protobuf/empty.proto";
 
 // MessageID is the unique identifier of a message.
 message MessageID {
-    bytes id = 1;
+    string id = 1;
```

Comment:
MessageID will be stored in wal's message properties.
so it should be binary safe represented by utf-8 string for compatibility.